hack/package-helm-charts.sh

```diff
@@ -19,12 +19,11 @@ set -o pipefail
 # if arg1 is set, it will be used as the version number
 if [ -z "$1" ]; then
   VERSION=$(awk -F= '/^VERSION/ { print $2 }' versions.mk | tr -d '[:space:]')
-  # Remove any v prefix, if exists.
-  VERSION="${VERSION#v}"
```

Comment:
Btw, all of this is a great example that (well-maintained) code comments easily can save a bunch of time overall.